build/includes/website.mk

```diff
@@ -25,12 +25,14 @@
 #
 
 # generate the latest website
+site-server: HUGO_ENV ?=snapshot
 site-server: ARGS ?=-F
 site-server: ENV ?= RELEASE_VERSION="$(base_version)" RELEASE_BRANCH=main
 site-server: ensure-build-image
 	docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) -p 1313:1313 $(build_tag) bash -c \
-	"$(ENV) hugo server --watch --baseURL=http://localhost:1313/ --bind=0.0.0.0 $(ARGS)"
+	"$(ENV) hugo server -e $(HUGO_ENV) --watch --baseURL=http://localhost:1313/ --bind=0.0.0.0 $(ARGS)"
 
+site-static: HUGO_ENV ?=snapshot
```

Comment:
Same as above - we have `ENV`, so not sure we need a `HUGO_ENV` setting?

Reply:
Ah OK, I didn't realize that Hugo recognizes all HUGO_* environment variables https://gohugo.io/getting-started/configuration/#configure-with-environment-variables. I'll go ahead and use the environment variable instead of the flag.